Description: add path for qtwaylandscanner
Author: Jonathan Riddell <jriddell@ubuntu.com>
Origin: upstream, https://git.reviewboard.kde.org/r/122526/
Last-Update: 2015-02-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/find-modules/FindQtWaylandScanner.cmake b/find-modules/FindQtWaylandScanner.cmake
index 28a4fda..b11b552 100644
--- a/find-modules/FindQtWaylandScanner.cmake
+++ b/find-modules/FindQtWaylandScanner.cmake
@@ -90,7 +90,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/ECMFindModuleHelpersStub.cmake)
 ecm_find_package_version_check(QtWaylandScanner)
 
 # Find qtwaylandscanner
-find_program(QtWaylandScanner_EXECUTABLE NAMES qtwaylandscanner)
+find_program(QtWaylandScanner_EXECUTABLE NAMES qtwaylandscanner PATHS /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}/qt5/bin/)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(QtWaylandScanner
